jetty-client/src/main/java/org/eclipse/jetty/client/DuplexHttpDestination.java

```diff
@@ -16,25 +16,17 @@
 //  ========================================================================
 //
 
-package org.eclipse.jetty.client.http;
+package org.eclipse.jetty.client;
 
-import org.eclipse.jetty.client.HttpClient;
-import org.eclipse.jetty.client.HttpExchange;
-import org.eclipse.jetty.client.Origin;
-import org.eclipse.jetty.client.PoolingHttpDestination;
-import org.eclipse.jetty.client.SendFailure;
-import org.eclipse.jetty.client.api.Connection;
-
-public class HttpDestinationOverHTTP extends PoolingHttpDestination
+public class DuplexHttpDestination extends HttpDestination
```

Comment:
I don't understand the name change? why `Duplex`? as it bidirectional?  Is there a Simplex one?   Javadoc would help.

Reply:
There were many `HttpDestination` subclasses before, but with the changes in this PR they have been reduce to only 2.
I added the javadocs.
"Duplex" because the transports create the destinations, and the transport knows if it is a duplex transport (e.g. HTTP/1.1 or FastCGI), or a multiplex transport (e.g. HTTP/2).

Reply:
It's Duplex and Simplex.  I don't think Duplex vs Multiplex works.  https://wikidiff.com/multiplex/duplex.  But I can't think of a better one....